phpBB/includes/ucp/ucp_main.php

```diff
@@ -152,6 +159,19 @@ function main($id, $mode)
 					$folder_img = ($unread_topic) ? $folder_new : $folder;
 					$folder_alt = ($unread_topic) ? 'UNREAD_POSTS' : (($row['topic_status'] == ITEM_LOCKED) ? 'TOPIC_LOCKED' : 'NO_UNREAD_POSTS');
 
+					// Replies
+					$replies = $phpbb_content_visibility->get_count('topic_posts', $row, $forum_id) - 1;
+
+					if ($topics_count)
```

Comment:
This currently does not do anything. It seems like it's aimed at having pagination for more topics than the topics per page. But it's not used for the query (so start doesn't do anything) and also nothing is displayed once topics count is actually not zero anymore.